tests/release/pipelines/fbc_release.go

```diff
@@ -325,6 +325,7 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 			"preGA":                           preGA,
 			"productName":                     productName,
 			"productVersion":                  productVersion,
+			"allowedPackages":                 []string{"example-operator"},
```

Comment:
`opm render` on the `containerImage` will return this? Maybe a question for @jinqi7 to chime in on. I am not too well versed in the `CreateComponentByCDQ` function

Reply:
I think it will - because it appears right here: https://github.com/redhat-appstudio-qe/fbc-sample-repo/blob/main/4.13/catalog/example-operator/catalog.yaml#L3

Reply:
Oh, perfect